### ISSUE TITLE
Outdated `TerseBitstringStatusListEntry` to `BitstringStatusListEntry` conversion

### ISSUE THREAD

**Issue by timothee-haudebourg** [w3c/vc-barcodes#19]
The new definition of `BitstringStatusListEntry` includes the `statusSize` and `statusMessages`, that cannot be derived from `TerseBitstringStatusListEntry`.

**Comment by msporny** [comment]
Yes, `statusSize` and `statusMessages` isn't supported in `TerseBitstringStatusListEntry` because it would bloat the size of the vc-barcode to the point where the VC wouldn't be very compact. We had to make some trade-offs with `TerseBitstringStatusListEntry` and this was one of them.

Does that create an issue for your use cases, @timothee-haudebourg ?

**Comment by timothee-haudebourg** [comment]
It does create an issue because in the newest version of Bitstring Status List, `statusSize` and `statusMessages` are stored in `BitstringStatusListEntry`. Here the `BitstringStatusListEntry` is supposed to be derived from `TerseBitstringStatusListEntry`, which does not support those properties.

**Comment by msporny** [comment]
> Here the `BitstringStatusListEntry` is supposed to be derived from `TerseBitstringStatusListEntry`, which does not support those properties.

Yes, correct, the `statusSize` and `statusMessages` properties are not supported.

Would adding text explicitly calling that out address your concern, @timothee-haudebourg? What concrete changes would you like to see made to the specification based on your comments above?

**Comment by timothee-haudebourg** [comment]
Sorry for the delay, I'm just coming back to this issue. I guess the only way to fix it is to either
- put the `statusSize` and `statusMessages` properties back into the `BistringStatusList` (but that's not the direction taken by the WG it seems); or
- pass them as input of [the `TerseBitstringStatusListEntry` conversion algorithm](https://w3c-ccg.github.io/vc-barcodes/#convert-status-list-entries). But then the verifier needs to know those parameters in advance.
